docs/management-ui.mdx

```diff
@@ -55,6 +55,16 @@ DashboardAccessOut dashboard = svix.getAuthentication().dashboardAccess("app_Xzx
 System.out.println(dashboard.getUrl());
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+val svix = Svix("AUTH_TOKEN")
+val dashboard: DashboardAccessOut = svix.authentication.dashboardAccess("app_Xzx8bQeOB1D1XEYmAJaRGoj0")
```

Comment:
No need to explicitly set the type of the var.

Reply:
Done.